td.gemspec

```diff
@@ -15,19 +15,19 @@ Gem::Specification.new do |gem|
   gem.test_files            = `git ls-files -- {test,spec,features}/*`.split("\n")
   gem.executables           = `git ls-files -- bin/*`.split("\n").map{ |f| File.basename(f) }
   gem.require_paths         = ['lib']
-  gem.required_ruby_version = '>= 2.0'
+  gem.required_ruby_version = '>= 2.1'
 
-  gem.add_dependency "msgpack", [">= 0.4.4", "!= 0.5.0", "!= 0.5.1", "!= 0.5.2", "!= 0.5.3", "< 0.8.0"]
+  gem.add_dependency "msgpack"
   gem.add_dependency "yajl-ruby", "~> 1.1"
   gem.add_dependency "hirb", ">= 0.4.5"
   gem.add_dependency "parallel", "~> 1.8.0"
-  gem.add_dependency "td-client", "~> 0.8.83"
-  gem.add_dependency "td-logger", "~> 0.3.21"
+  gem.add_dependency "td-client", "~> 0.8.84"
+  gem.add_dependency "td-logger", ">= 0.3.21", "< 2"
```

Comment:
Yeah, 1.0 has come now and it looks compatible with 0.3.21; but 2.x is far future and no one knows what it is.